templates/.VERSION

```diff
@@ -1 +1 @@
-0.9.99.0915
\ No newline at end of file
+v0.9.99-212-g2c92e45
```

Comment:
I think the .VERSION file is there as a kind of "safety measure" to avoid using a set ot templates meant for a different version of the built gitea. If we want to obtain the same safety we should still include a version but maybe keep it separated from the application version (a template API version). Could easily become hard to maintain though.

Or, we give up on the attempt and just print a WARNING whenever templates/ are not embedded (see #tboerger PR about embedding template/ and public/)